views/default/elements/forms/footer.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * Wrap form footer
+ * @uses $vars['footer']      Form footer
+ * @uses $vars['action_name'] Action name
+ */
+$footer = elgg_extract('footer', $vars);
+if (empty($footer)) {
+	return;
+}
+
+echo elgg_format_element('div', [
+	'class' => 'elgg-foot elgg-form-footer',
```

Comment:
Too much hassle trying to get those vars to the view. We would have to store them somewhere until we render the footer.